### PR TITLE
feat: Remove search time activity with fragment using navigation

### LINKED ISSUE #822: Make single activity app with navigation architecture component
To Make a single activity app - 
- [x] For navigation fragments and profile fragments as main activity host. PR #823 
- [x] For all root layout fragments. PR #831 
- [x] Remove search results activity. PR #839 
- [x] Handle authentication with main activity and remove authentication activity. PR #844 
- [x] Replace search time activity with fragment. PR #851 
- [ ] Replace search location activity with fragment.

**Would you like to work on the issue?**

- [x] Yes

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #822

### CODE REVIEW

File: app/src/main/res/layout/fragment_search_time.xml [hunk 1]
```diff
@@ -69,5 +70,5 @@
         android:text="@string/pick_date"
         android:textColor="@color/light_grey"
         android:textSize="@dimen/text_size_expanded_title" />
-
 </LinearLayout>
+
```
Comment (nikit19) [inline app/src/main/res/layout/fragment_search_time.xml]: Extra

Comment (liveHarshit) [reply app/src/main/res/layout/fragment_search_time.xml]: But this new link is automatically added at the and of layout when I commit the file.

Comment (liveHarshit) [reply app/src/main/res/layout/fragment_search_time.xml]: How to complies play store package in android studio? Please help build is showing error in that package.

Comment (nikit19) [reply app/src/main/res/layout/fragment_search_time.xml]: Choose the playstore flavor and then build the app.

Comment (liveHarshit) [reply app/src/main/res/layout/fragment_search_time.xml]: Got it. Thanks :smile:

**Comment by liveHarshit** [comment]
Updated. Build successful.

**Pull request merged**